authentication/authentication-styles.css

```diff
@@ -0,0 +1,76 @@
+body {
+    margin-left: 20px;
+    margin-right: 20px;
+    margin-top: 100px;
+    margin-bottom: 50px;
+}
+
+.card {
+    max-width: 50vw;
+    min-width: fit-content;
+    height: 50vh;
+    background-color: white;
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    margin: auto;
+    border-radius: 10px;
+    top: 50%;
+    box-shadow: 5px 5px 10px grey;
+
+}
+
+p {
+    width: 80%;
```

Comment:
Instead of setting width, how about setting the `margin-left` & `margin-right` for the text?